scripts/release/utils.injector.js

```diff
@@ -0,0 +1,249 @@
+ * @returns {string} text comment
+ */
+const getComment = (signature) => {
+  if (signature?.comment?.summary?.length < 0) return '';
+  return signature?.comment?.summary.map((item) => item.text).join('');
+};
+
+/**
+ * Return full Return text from Signature Typedoc structure.
+ * @param signature type from typedoc
+ * @returns {string} Return comment
+ */
+const getReturnComment = (signature) => {
+  const blockTags = signature?.comment?.blockTags;
+  const contents = blockTags?.find((item) => item.tag === '@returns')?.content || [];
+  return contents.map((item) => item.text).join('');
+};
+
+/**
+ * Generate and return Parameter table in structure of json2md.
+ * @param params array parameters of { name, type, description }
+ * @returns {array} json2md array
+ */
+const generateParameter = (params) => {
+  const result = [];
+  result.push({ h4: 'Arguments' });
+
+  const table = {
+    table: {
+      headers: ['Name', 'Type', 'Description'],
+      rows: []
+    }
+  };
+  if (params.length > 0) {
+    for (const { name, type, description } of params) {
+      table.table.rows.push([name || '', type || '', sanitize(description) || '']);
+    }
+  }
+  result.push(table);
+  return result;
+};
+
+/**
+ * Generate and return Return table in structure of json2md.
+ * @param obj object which has properties type and description
+ * @returns {array} json2md array
+ */
+const generateReturn = ({ type, description }) => {
+  const result = [];
+  result.push({ h4: 'Returns' });
+  const table = {
+    table: {
+      headers: ['Type'],
+      rows: [[type]]
+    }
+  };
+  if (description) {
+    table.table.headers.push('Description');
+    table.table.rows[0].push(sanitize(description));
+  }
+  result.push(table);
+  return result;
+};
+
+/**
+ * Generate and return Constructor content in structure of json2md.
+ * @param IDs array of id that id is type which matched by typedoc
+ * @param dataClass Class that used to convert to md
+ * @returns {array} json2md array
+ */
+const generateConstructor = (IDs, dataClass) => {
+  const result = [];
+  const data = dataClass.children.find((item) => item.id === IDs[0]);
+  if (!data) return result;
+
+  result.push({ h2: 'Constructor' });
+  if (data?.signatures[0].parameters?.length > 0) {
+    const params = data?.signatures[0].parameters.map((item) => {
+      return {
+        name: item?.name,
+        type: item?.type?.name,
+        description: getComment(item)
+      };
+    });
+    result.push(...generateParameter(params));
+  }
+  return result;
+};
+
+/**
+ * Generate and return Constructor content in structure of json2md.
+ * @param IDs array of id that id is type which matched by typedoc
+ * @param dataClass Class that used to convert to md
+ * @returns {array} json2md array
+ */
+const generateAccessor = (IDs, dataClass, mappedSignatures) => {
+  const result = [];
+  if (IDs?.length < 0) return result;
+  result.push({ h2: 'Accessors' });
+  for (const id of IDs) {
```

Comment:
what would happen if `IDs.length` is 0?

Reply:
Fixed. Should don't do if `IDs.length` is 0